packages/ast-manipulation/js/common.ts

```diff
@@ -57,3 +57,41 @@ export function addFromString(ast: AstTypes.BlockStatement | AstTypes.Program, v
         ast.body.push(childNode);
     }
 }
+
+export function expressionFromString(value: string): AstKinds.ExpressionKind {
+    const program = parseScript(value);
+    const statement = program.body[0];
+    if (statement.type !== "ExpressionStatement") {
+        throw new Error("value passed was not an expression");
+    }
+
+    return statement.expression;
+}
+
+export function statementFromString(value: string): AstKinds.StatementKind {
+    const program = parseScript(value);
+
+    return program.body[0];
```

Comment:
In order to provide consistency with the method above, shouldn't we also check the generated statement type or throw an exception if anything other was found?

I think something like this should work:
```js
const program = parseScript(value);
const statement = program.body[0];
if (statement.type && statement.type.includes("statement")) {
    throw new Error("value passed was not an statement");
}

return statement.expression;
```

Reply:
Trying it out, I don't think validating it will work. For instance, a variable declaration is a statement, but its `type` is `VariableDeclaration`, which doesn't contain the word `Statement`. Perhaps just checking if the value `program.body[0]` exists is good enough for this

Reply:
Sound good!